content/sensu-go/5.16/api/filters.md

```diff
@@ -183,7 +240,7 @@ payload         | {{< highlight shell >}}
   },
   "action": "deny",
   "expressions": [
-    "event.entity.metadata.namespace == 'production'"
+    "event.entity.metadata.namespace == '\''production'\''"
```

Comment:
Also, this is stating that we're looking for a namespace called 'production', but looks like we're creating a 'development' filter...should we match this so that namespace == dev?

Reply:
yes, I think you're right, I'll fix that! Thank you.